repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/commands/VerifyTryItNowCommand.scala

```diff
@@ -487,6 +507,23 @@ object VerifyTryItNowCommand extends Command {
     }
   }
 
+  private def describeFoyerStatus(foyerStatus: FoyerStatus.FoyerStatus): String = {
+    foyerStatus match {
+      case FoyerStatus.Allowed =>
+        "Users with the 'foyer' Identity role WILL pass through this component"
+      case FoyerStatus.AllowedAuthorized =>
+        "Users with the 'foyer' Identity role WILL pass through this component IF AND ONLY IF their Identity " +
+          "service catalog contains an endpoint required by the authorization component"
+      case FoyerStatus.AllowedNotAuthorized =>
+        "Users with the 'foyer' Identity role WILL pass through this component BUT authorization checks will not " +
+          "be performed"
+      case FoyerStatus.NotAllowed =>
+        "Users with the 'foyer' Identity role WILL NOT pass through this component"
+      case FoyerStatus.Unknown =>
+        "Users with the 'foyer' Identity role MAY OR MAY NOT pass through this component"
+    }
+  }
+
   object FoyerStatus extends Enumeration {
     type FoyerStatus = Value
     val Allowed, AllowedAuthorized, AllowedNotAuthorized, NotAllowed, Unknown = Value
```

Comment:
`AllowedNotAuthorized` should have probably been `AllowedNoAuthorization`, but the comments and accompanying descriptions spell it out clear enough.